lib/cocoapods/target/aggregate_target.rb

```diff
@@ -222,17 +222,17 @@ def framework_paths_by_config
       framework_paths_by_config
     end
 
-    # @return [Hash{ Symbol => Array<Pathname> }] Uniqued Resources grouped by config
+    # @return [Hash{ Symbol => Array<String> }] Uniqued Resources grouped by config
     #
-    def resources_by_config
+    def resource_paths_by_config
       library_targets = pod_targets.reject do |pod_target|
         pod_target.should_build? && pod_target.requires_frameworks?
       end
       user_build_configurations.keys.each_with_object({}) do |config, resources_by_config|
         resources_by_config[config] = library_targets.flat_map do |library_target|
           next [] unless library_target.include_in_build_config?(target_definition, config)
           resource_paths = library_target.file_accessors.flat_map do |accessor|
-            accessor.resources.flat_map { |res| res.relative_path_from(sandbox.project.path.dirname) }
+            accessor.resources.flat_map { |res| "${PODS_ROOT}/#{res.relative_path_from(sandbox.project.path.dirname)}" }
```

Comment:
This was slightly a mistake. This was returning `Pathname` versus the one on line 238 was returning Strings.

Furthermore, we should be prepending `${PODS_ROOT}` here.